pydis_site/apps/content/resources/security-notice.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Security Notice
+description: How vulnerabilities in our projects should be reported.
+icon: fas fa-lock
```

Comment:
The lock icon for the security page may be misleading - my first thought was that it was an inaccesible page. Do you think a shield icon would be better?

Reply:
Addressed in 5c44a33.